pkg/specgen/namespaces.go

```diff
@@ -215,12 +245,17 @@ func ParseUserNamespace(ns string) (Namespace, error) {
 func ParseNetworkNamespace(ns string) (Namespace, []string, error) {
 	toReturn := Namespace{}
 	var cniNetworks []string
+	// Net defaults to Slirp on rootless
+	if rootless.IsRootless() {
+		toReturn.NSMode = Slirp
+		return toReturn, nil, nil
+	}
 	switch {
 	case ns == "slirp4netns":
 		toReturn.NSMode = Slirp
 	case ns == "pod":
 		toReturn.NSMode = FromPod
-	case ns == "bridge":
+	case ns == "bridge", ns == "":
```

Comment:
I think we need a separate `case` for `""` where we check if we are rootless and return `Bridge` or `Slirp` based on that

Reply:
Yes, you are right fixed.